Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-settings.jsp

```diff
@@ -8,27 +8,31 @@
 <script
 	src="${pageContext.servletContext.contextPath}/resources/txt-layout/js/jstree.min.js"></script>
 
-
 <div>
-    <ul>
-        <li><a
-            href="${pageContext.servletContext.contextPath}/auth/workbench/projects/${project.projectId}"><span
-                class="glyphicon glyphicon-circle-arrow-left"></span> Back to Project
-        </a></li>
-    </ul>
+    <a href="${pageContext.servletContext.contextPath}/auth/workbench/projects/${project.projectId}"><span
+                class="glyphicon glyphicon-circle-arrow-left"></span> Back to Project</a>
 </div>
 
-<h2 class="major" href="${pageContext.servletContext.contextPath}/auth/workbench/${project.projectId}/settings">
-	<span>Settings</span>
-</h2>
-<a
-    href="${pageContext.servletContext.contextPath}/auth/workbench/projects/${project.projectId}/settings"><i class="fa fa-cog"></i> Main Page Settings</a>
+<br/>
+
+<div class="panel panel-default">
+  
+    <div class="panel-heading">
+    <h3 class="panel-title">Settings</h3>
+    </div>
+    
+    <div class="list-group">
+    <a class="list-group-item"
```

Comment:
the items are never active, are they? Depending on which page a user is, the corresponding menu entry should be active.